packages/wonder-blocks-modal/src/components/one-pane-dialog.stories.js

```diff
@@ -200,3 +201,41 @@ WithOpener.parameters = {
         disableSnapshot: true,
     },
 };
+
+export const WithClosedFocusId: StoryComponentType = () => {
```

Comment:
Could you please add the `disableSnapshot` config here? This story doesn't need to be to be sent to Chromatic/visual testing because the thing we want to test doesn't appear on first render, but it happens until we complete a series of steps.